### ISSUE TITLE
Did you keep the best checkpoint in the unsupervised setting?

### ISSUE THREAD

**Issue by IkutoSha** [princeton-nlp/SimCSE#59]
Hi,
in Appendix A, you said that "We evaluate the model every 250 training steps on the development set of STS-B and keep the best checkpoint for the final evaluation on test sets. For unsupervised SimCSE, we take 5e-5 as the learning rate for both base and large models and only train for one epoch." I want to know that using evaluation on the development set of STS-B is just on the supervised setting or both supervised and unsupervised settings. I found that in run_unsup_example.sh there are
 `--evaluation_strategy steps \
  --metric_for_best_model stsb_spearman \
  --load_best_model_at_end \
  --eval_steps 125 \`
and in trainers.py there are 
`if self.args.load_best_model_at_end and self.state.best_model_checkpoint is not None:
            logger.info(
                f"Loading best model from {self.state.best_model_checkpoint} (score: {self.state.best_metric})."
            )`

**Comment by gaotianyu1350** [comment]
Hi,

For both models, we take the same best-checkpoint policy.

**Comment by IkutoSha** [comment]
Thanks a lot.
